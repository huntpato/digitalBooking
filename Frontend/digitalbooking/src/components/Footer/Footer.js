import React, { Fragment } from "react";
import styles from "./Footer.module.css";

export const Footer = (props) => {
  const { footer, social } = styles;

  return (
    <Fragment>
      <footer className={footer}>
        <h6>Â©2021 Digital Booking</h6>
        <div className={social}>
          <i className="fab fa-facebook"></i>
          <i className="fab fa-linkedin-in"></i>
          <i className="fab fa-twitter"></i>
          <i className="fab fa-instagram"></i>
        </div>
      </footer>
    </Fragment>
  );
};
