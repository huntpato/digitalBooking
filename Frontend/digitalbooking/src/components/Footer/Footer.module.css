footer{
    align-items: center;
    background-color: var(--principal-color);
    bottom: 0;
    color: var(--background-white);
    display: flex;
    flex-direction: row;
    justify-content: space-between;
    padding: 15px 50px 15px 40px ;
    width: 100%;
    height: 7vh;
    /**/
}

footer h6{
    color: var(--color-fondo2);
    margin-left: 2%;
}

.social{
    padding-right: 25px;
}

.social i{
    color: var(--color-fondo2);
    margin: 0px 10px 0px 10px ;
}

@media (max-width: 768px){
    
    footer h6{
        margin-left: 3%;
    }
    
    .social{
        padding-right: 10px;
    }
 
}

@media (max-width: 480px){

    .footer{
        padding: 15px 30px 15px 30px ;
    }

    footer h6{
        margin-left: 0%;
    }

    .social{
        padding-right: 0px;
    }

}