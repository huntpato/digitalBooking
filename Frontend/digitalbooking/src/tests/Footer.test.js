import React from 'react';
import { shallow } from 'enzyme';
import { Footer } from '../components/Footer/Footer'
import '@testing-library/jest-dom'
import { expect } from '@jest/globals';

describe('Pruebas en componente <Footer />', () => {
    const wrapper= shallow(<Footer/>);
   
    test('Comprobar que <Footer /> renderiza correctamente.', () => {

        expect( wrapper ).toMatchSnapshot();
    })

    test('Verificar que el componente carga la clase de CSS correctamente', () => {

        const footer = wrapper.find('footer');
        expect(footer.hasClass('footer')).toBe(true);
    })
    
    
})
