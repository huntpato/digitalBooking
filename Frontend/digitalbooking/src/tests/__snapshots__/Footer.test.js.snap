// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en componente <Footer /> Comprobar que <Footer /> renderiza correctamente. 1`] = `
<Fragment>
  <footer
    className="footer"
  >
    <h6>
      Â©2021 Digital Booking
    </h6>
    <div
      className="social"
    >
      <i
        className="fab fa-facebook"
      />
      <i
        className="fab fa-linkedin-in"
      />
      <i
        className="fab fa-twitter"
      />
      <i
        className="fab fa-instagram"
      />
    </div>
  </footer>
</Fragment>
`;
